tests/tests/Data/RawCodeCoverageDataTest.php

```diff
@@ -417,6 +417,9 @@ public function testReturnStatementWithConstantExprOnlyReturnTheLineOfLast(): vo
                 226,
                 235,
                 244,
+                250,
+                254,
```

Comment:
line 255 is variable as well and reported by xdebug, so it should be marked as executable/covered as well